qiita_db/support_files/patches/44.sql

```diff
@@ -0,0 +1,19 @@
+-- Dec 3, 2016
+-- Modify qiita.study_publication so studies can have string
+-- dois and pubmed ids
+
+
+-- dropping PRIMARY KEY ( study_id, publication_doi )
+ALTER TABLE qiita.study_publication DROP CONSTRAINT idx_study_publication_0;
```

Comment:
Thought about it and thought that study and is_doi should be it but at the same time I feel like it's not necessary, thus why I didn't add it. Please let me know if you feel strong about it.